lib/cartopy/io/img_tiles.py

```diff
@@ -682,3 +682,21 @@ def _image_url(self, tile):
             f'https://atlas.microsoft.com/map/tile?'
             f'api-version={self.api_version}&tilesetId={self.tileset_id}&'
             f'x={x}&y={y}&zoom={z}&subscription-key={self.subscription_key}')
+
+
+class LINZMapsTiles(GoogleWTS):
+    
+    def __init__(self, subscription_key, layer_id, api_version="v4",
```

Comment:
Could you add a description of these and how to obtain a `subscription_key` / link to the main landing page?

Reply:
Hi Greg,

Obtaining a subscription key/API key can be achieved by signing up at https://id.koordinates.com/signup/
(it is not a subscription service per se, just a unique API key for each registered user).
Layer ID is in the "Technical details" section of the "About" tab for each data in the LINZ data service (e.g. https://data.linz.govt.nz/layer/51769-auckland-05m-rural-aerial-photos-2010-2012/ , scroll down for details).
API version is currently v4 and determined manually from the WMTS descriptor.

Fian

Reply:
Thanks! I meant to add that information to the docstring so others can know about how to use this too :)

As a note: Someone with admin access on Cartopy should sign up for some of these services if possible so we can add the keys as secrets to the runners...

Reply:
Ahh, I see...

I am a bit of a newbie on this Github business, can just let me know if I need to comment on the .py file :-D
All links I have given here are open and do not need any login, however, the contents of WMTS GetCapabilities etc. are for registered users only.